node/actors/network/src/gossip/tests/mod.rs

```diff
@@ -1,6 +1,6 @@
 use super::*;
 use crate::{metrics, preface, rpc, testonly};
-use anyhow::Context as _;
```

Comment:
ditto

Reply:
I think that `context` is not being used in this module.

Reply:
I think that `context` is no used here, should we keep it?

Reply:
if it is unused, then yeah, I thought it was accidental.